jordan_bundy/public/index.html

```diff
@@ -0,0 +1 @@
+<!doctype html><html><head></head><body><h1>Hello World</h1><a href="/">/</a><br><a href="/name">/name</a><br><a href="/time">/time</a><br></body></html>
```

Comment:
Only minified because it's added from the javascript, so i didn't include line breaks or styling